index.src.html

```diff
@@ -972,8 +1081,8 @@ <h4 algorithm id="construct-passwordcredential-form">
                         result of executing |formData|'s {{FormData/get()}} method
                         on |name|, and |idName| to |name|.
 
-    7.  Let |c| be the result of executing the {{PasswordCredential(data)}} constructor on |data|.
-        Rethrow any exception generated.
+    7.  Let |c| be the result of executing <a abstract-op>Create a `PasswordCredential` from
```

Comment:
add before 7 that we throw a TypeError exception if we did not find a name/password (like in line 1121)?